src/cli/cli.cpp

```diff
@@ -76,12 +77,19 @@ TimeControl::Limits parseTc(const std::string &tcString) {
     }
 
     if (has_inc) {
-        const auto inc      = str_utils::splitString(remainingStringVector, '+');
+        const auto inc        = str_utils::splitString(remainingStringVector, '+');
         tc.increment          = static_cast<uint64_t>(std::stod(inc[1]) * 1000);
         remainingStringVector = inc[0];
     }
-
-    tc.time = static_cast<int64_t>(std::stod(remainingStringVector) * 1000);
+  
+    if (has_minutes) {
+        const auto minutes_string = str_utils::splitString(remainingStringVector, ':');
+        int64_t minutes = static_cast<int64_t>(std::stod(minutes_string[0]) * 1000);
+        int64_t seconds = static_cast<int64_t>(std::stod(minutes_string[1]) * 1000);
+        tc.time = minutes * 60 + seconds;
```

Comment:
the names are confusing btw, because minutes isn't really minutes anymore

Reply:
ahh yes, i should have multiplied it by 1000 after the variable is initialized

Reply:
fixed 👍